src/pages/ThorChainLP/components/ReusableLpStatus/TransactionRow.tsx

```diff
@@ -168,14 +189,39 @@ export const TransactionRow: React.FC<TransactionRowProps> = ({
 
     const pairedAddress = pairAssetAddress ?? ''
 
+    const asymDestinationPoolAssetId = (() => {
+      let poolAssetId: string | undefined
+
+      if (isRuneTx && opportunityType !== 'sym')
+        poolAssetId = assetIdToPoolAssetId({ assetId: thorchainAssetId })
+      if (!isRuneTx && opportunityType !== 'sym') poolAssetId = assetIdToPoolAssetId({ assetId })
+      if (opportunityType === AsymSide.Asset && type === 'sym')
+        poolAssetId = assetIdToPoolAssetId({ assetId })
+
+      if (poolAssetId) {
+        return poolAssetId
+      }
+    })()
```

Comment:
not sure if something changed after you added some of this code in here, but `assetId` will relate to the side for asym withdrawals
```suggestion
    const asymDestinationPoolAssetId = opportunityType !== 'sym' ? assetIdToPoolAssetId({ assetId }) : undefined
```

Reply:
Makes sense for sure! The code evolved after this has been added and it seems it got out of my radar!

Nice catch, fixed in [b602e31](https://github.com/shapeshift/web/pull/7351/commits/b602e31990567afc87ae66b6a0c081c5b9bbdb26)